base/dcp_common.go

```diff
@@ -58,10 +58,12 @@ type DCPCommon struct {
 	feedID                 string                         // Unique feed ID, used for logging
 	loggingCtx             context.Context                // Logging context, prefixes feedID
 	checkpointPrefix       string                         // DCP checkpoint key prefix
+	janitorRollback        func()                         // This function will trigger a janitor_pindex_rollback
```

Comment:
I thought this should be stored on DCPDest, since it's a cbgt-specific artifact, but I guess DCPDest is actually the only usage of DCPCommon now that we've switched the other feeds over to DCPClient?